go.mod

```diff
@@ -4,13 +4,12 @@ go 1.18
 
 require (
 	github.com/fatih/color v1.12.0 // indirect
-	github.com/ghodss/yaml v1.0.0 // indirect
 	github.com/gogo/protobuf v1.3.2
 	github.com/google/go-cmp v0.5.6 // indirect
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0 // indirect
 	github.com/prometheus/client_golang v1.11.1
 	github.com/stretchr/testify v1.7.5
-	github.com/vmware-tanzu/carvel-vendir v0.24.0
+	github.com/vmware-tanzu/carvel-vendir v0.29.1-0.20220727190601-9dd41a08ed60
```

Comment:
This fairly large fast forward causes us to pull in more vendored things, hence why there are so many file changes...